src/exercises/exerciseTypes/Exercise.sc.js

```diff
@@ -60,6 +60,53 @@ const Exercise = styled.div`
   }
 `;
 
+let MatchInputHolder = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+`;
+
+let MatchButtonHolder = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+let MatchSpeakButtonHolder = styled.div`
+  width: fit-content;
+  margin: 2em;
+  margin-left: -0.75em;
+  border-radius: 0.75em;
+  padding: 0.5em;
+`;
+
+let MatchButton = styled.button`
+  width: fit-content;
+  margin: 2em;
+  border-radius: 0.75em;
+  padding: 0.5em;
+  background: #ffd04799;
+  color: black;
+  border: 0.125em solid #ffbb54;
+  border-style: none;
+  box-shadow: none;
+  user-select: none;
+
+  font-weight: 500;
+  cursor: pointer;
+  font-family: "Montserrat", -apple-system, BlinkMacSystemFont, "Segoe UI",
```

Comment:
It feels wrong to define fonts at this point in the definition. 
Can't we use the generic project fonts? 

In general, the MatchButton defines very low-level styles, 
that probably should be defined in a more generic button.
E.g. box-shadow? Also, I see that the Submit button has 
also a lot of duplicate styling. At least between these two
we should factor out the comonalities; but as I said, there
should probably be a more common ancestor of these 
buttons.

Reply:
Avoided button duplication by creating common ancestor (`StyledButton`) and added the `font family` to `button` and `input` (until now only for `body` in `index.css`). See commit fca2e0c148b12bc6f78c8591952c1d99ac95ddfd